tequilapi/endpoints/access_policies.go

```diff
@@ -68,7 +68,7 @@ func NewAccessPoliciesEndpoint(httpClient *requests.HTTPClient, accessPolicyEndp
 //   500:
 //     description: Internal server error
 //     schema:
-//       "$ref": "#/definitions/ErrorMessageDTO"
+//       "$ref": "#/definitions/APIError"
```

Comment:
What about other endpoints?

Reply:
Looks like this is getting a positive feedback so far, hence I’ll update the responses for other endpoints as well